web/lib/components/flashcards/PageAllSets.jsx

```diff
@@ -21,6 +57,40 @@ export function PageAllSets() {
     <Layout>
       <div className="container">
         <Sets flashcardSets={flashcardSets} />
+        <div className="mt-5 flex w-full flex-row items-center justify-center gap-5">
+          <Button
+            onClick={prev}
+            disabled={currentPage === 0}
+            className="mr-3 inline-flex items-center rounded-lg border border-gray-300 bg-white py-2 px-4 text-sm font-medium text-gray-500 hover:bg-gray-100 hover:text-gray-700 dark:border-gray-700 dark:bg-gray-800 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-white">
```

Comment:
`Button` now has `variant` so it may not desired to pass `className`. Either use normal `button` or some variant from `Button`.